libraries/RC_Channel/RC_Channel.h

```diff
@@ -173,7 +173,7 @@ class RC_Channel {
         DRIFT     =           73, // drift mode
         SAILBOAT_MOTOR_3POS = 74, // Sailboat motoring 3pos
         SURFACE_TRACKING =    75, // Surface tracking upwards or downwards
-        STAND_BY  =           76, // Standy signal magnitude
+        STAND_BY  =           76, // Standy mode
```

Comment:
Shouldn't this be:
` STANDBY  =           76, // Standy mode`

Reply:
And why is it "Standy" in the comments? :-)

Reply:
Fixed the comment.

Canonicalised on `STANDBY` rather than `STAND_BY`.